api-server/src/server/boot/certificate.js

```diff
@@ -337,7 +337,7 @@ function createShowCert(app) {
   return function showCert(req, res, next) {
     let { username, cert } = req.params;
     username = username.toLowerCase();
-    const certType = superBlockCertTypeMap[cert];
+    const certType = certSlugTypeMap[cert];
```

Comment:
@ShaunSHamilton, [cert] should be [certSlug]. The relative naming on the certification page and showCertification component, etc should be adjusted accordingly. Although it could be resolved in another pr.

Reply:
Thanks, for catching this. I have pushed the changes.